tools/sdk-testgen/ci.yml

```diff
@@ -1,126 +1,127 @@
-
-          - script: |
-              commitId=$(git rev-parse --short HEAD)
-              rush publish --apply --prerelease-name="ci.$commitId"
-            displayName: Bump versions
-            workingDirectory: $(Build.SourcesDirectory)/tools/sdk-testgen
-          - script: |
-              rush publish --publish --pack --include-all
-            displayName: "Pack packages"
-            workingDirectory: $(Build.SourcesDirectory)/tools/sdk-testgen
-          - publish: $(Build.SourcesDirectory)/tools/sdk-testgen/common/temp/artifacts/packages
-            artifact: packages
-            displayName: Publish packages(.tgz) as pipeline artifacts
-      - job: Testmodeler
-        displayName: "TestModeler"
-        steps:
-          - script: |
-              npm ci
-            displayName: "npm ci"
-            workingDirectory: $(Build.SourcesDirectory)/tools/sdk-testgen/packages/autorest.testmodeler
-          - script: |
-              npm run test
-            displayName: "npm run test"
-            workingDirectory: $(Build.SourcesDirectory)/tools/sdk-testgen/packages/autorest.testmodeler
-
-  - ${{if eq(variables['Build.Reason'], 'Manual')}}:
-    - stage: release
-      displayName: "Release Package"
-      dependsOn: build
-      condition: succeeded()
-      jobs:
-        - job: approve
-          pool: server
-          displayName: "Approve"
-          steps:
-          - task: ManualValidation@0
-            inputs:
-              notifyUsers: 'Click to approve if it''s an expected public release.'
-        - job: release
-          displayName: "Pack and Release"
-          dependsOn: approve
-          condition: and(succeeded(), ne(variables['USER_SKIP_PUBLIC_RELEASE'], 'true'))
-          steps:
-          - template: ../../eng/pipelines/templates/steps/sdk-testgen-set-env.yml
-          - script: |
-              rush rebuild -v
-            displayName: "Build all packages"
-            workingDirectory: $(Build.SourcesDirectory)/tools/sdk-testgen
-          - script: |
-              rush publish --publish --pack --include-all
-            displayName: "Pack packages"
-            workingDirectory: $(Build.SourcesDirectory)/tools/sdk-testgen
-          - script: |
-              export NPM_AUTH_TOKEN="$(azure-sdk-npm-token)"
-              rush publish --publish --include-all --set-access-level public
-            displayName: Publish packages
+# Node.js
+# Build a general Node.js project with npm.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+trigger:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - tools/sdk-testgen
+
+pr:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - tools/sdk-testgen
+
+pool:
```

Comment:
This pool is the actual change. Not sure why entire file shows as diffed, perhaps due to automatic line endings conversion.